tests/desktop/consumer_pages/test_users_account.py

```diff
@@ -105,7 +105,8 @@ def test_recommended_tab_shows_up_only_if_checkbox_is_selected(self, mozwebqa, n
 
         Assert.true(basic_info.is_recommended_tab_visible)
 
-        basic_info.toggle_recommendations()
+        basic_info.disable_recommendations()
         basic_info.save_changes()
+        basic_info.wait_for_recommended_tab_not_visible()
 
         Assert.false(basic_info.is_recommended_tab_visible)
```

Comment:
The line before makes this redundant, but I'm fine with keeping it for test clarity.